internal/resolver/testdata/test-project2/mtaBad.yaml

```diff
@@ -0,0 +1 @@
+bad bad bad very bad
```

Comment:
This file isn't in use in the tests (unless I missed something). Please remove it.

Reply:
Done